notebooks/official/model_monitoring/get_started_with_model_monitoring_automl.ipynb

```diff
@@ -31,7 +31,7 @@
         "id": "fsv4jGuU89rX"
```

Comment:
<p>What <code>pre-trained model</code>? Isn't this notebook training a new model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
done